generated/opentxs.hpp.in

```diff
@@ -3,8 +3,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-#ifndef OPENTXS_OPENTXS_HPP
-#define OPENTXS_OPENTXS_HPP
+#pragma once
```

Comment:
The "guidelines" used in an attempt to justify the abuse of lambdas explicitly warn against using this non-portable non-standard pragma.